### PR TITLE
Add background-repeat to safe_style_css

### PR DESCRIPTION
**Pull request by andrewserong**
This PR adds the `background-repeat` property to `safe_style_css` to support the missing background repeat property.

This is a prerequisite for adding in a background-repeat toggle to the background image support feature, which has landed in Gutenberg in: https://github.com/WordPress/gutenberg/pull/57005

Note: that PR includes other PHP changes that will be backported separately, so that this PR can just focus on the proposed change to `kses.php` and its tests.

To test, ensure the tests pass, or if you're using the Docker environment for wordpress-develop, run the following locally: npm run test:php /var/www/tests/phpunit/tests/kses.php

Trac ticket: https://core.trac.wordpress.org/ticket/60132

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by ramonjd — APPROVED** [review]
I ran the tests. 

The tests passed.

👍🏻

**Comment by andrewserong** [comment]
Thanks for giving this a quick look! I forgot to add the ticket number to the test that uses this data provider. Added in https://github.com/WordPress/wordpress-develop/pull/5814/commits/5fc84d2f977ce0bd3016660ceab2d03f664203d6

File: tests/phpunit/tests/kses.php [hunk 1]
```diff
@@ -938,6 +938,7 @@ public function test_wp_kses_attr_no_attributes_allowed_with_false() {
 	 * @ticket 55966
 	 * @ticket 56122
 	 * @ticket 58551
+	 * @ticket 60132
 	 * @dataProvider data_safecss_filter_attr
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/kses.php]:
```suggestion
	 * @ticket 60132
	 *
	 * @dataProvider data_safecss_filter_attr
```

Nit-pick

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r57228](https://core.trac.wordpress.org/changeset/57228).

**Pull request closed without merging**